scripts/dog-image.js

```diff
@@ -0,0 +1,96 @@
+
+function murphy() {
+  let dogName = ['Murphy'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", Great Dane, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function poppy() {
+  let dogName = ['Poppy'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", Maltese, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function jack() {
+  let dogName = ['Jack'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", Beagle, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function duffy() {
+  let dogName = ['Duffy'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", American Staffordshire Terrier, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function lucas() {
+  let dogName = ['Lucas'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", Shepherd Mix, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function jake() {
+  let dogName = ['Jake'];
+  const dogPrice = 123.45;
+  const dogResult = dogName + ", yellow lab, costs $" + String(dogPrice) + ' to adopt.';
+
+  alert(dogResult);
+}
+
+function angus() {
```

Comment:
For all of these dog  tiles, you'll want to use one function, then pass in the information from the HTML. Something like this..https://codepen.io/jimmynono/pen/JqrZeq